src/inmanta/plugins.py

```diff
@@ -558,3 +568,16 @@ def wrapper(self, *args: object, **kwargs: object) -> Any:
     elif function is not None:
         fnc = curry_name(commands=commands, emits_statements=emits_statements, allow_unknown=allow_unknown)
         return fnc(function)
+
+
+@stable_api
+def deprecated(
+    function: Optional[Callable] = None, replaced_by: Optional[str] = None, **kwargs: Dict[str, object]
```

Comment:
```suggestion
    function: Optional[Callable] = None, *, replaced_by: Optional[str] = None, **kwargs: Dict[str, object]
```
No positional args other than `function`, remember? Otherwise our lambda is not compatible.

Reply:
right, I always forget about the *. thanks :p